### ISSUE TITLE
Relationship FU Plan: Mike Soltys

### ISSUE THREAD

**Issue by jaahmuhl** [promazo/100k-Mentor-Challenge-Outreach#144]
**User Story / Problem / Details**
The Content Editor is in charge of ensuring the initial relationship we foster with an interviewee does not decay over time for interviewees that we designate as important. As a result we want to follow a process that allows us to interact and build a relationship with the interviewee over time. The final step of the process should be focused on how connect with an interviewee over multiple interactions and over the course of a year.

**Technical Definition of Done**
Complete all steps in the process

**Definition of Ready**
All of the steps in the process have been approved by Brett 

**Process**
- [ ] Review the interview in its entirety 
- [ ] Thank you note to the participant
- [ ] Quick clip (30s) that shows the interviewee some of the raw footage/a particularly interesting part of the discussion
- [ ] ID potential contacts
- [ ] Formulate ask
- [ ] Create a Relationship Action Plan

**Comment by howie-emond** [comment]
Hi Mike, we wanted to reach out and thank you again for participating in our Executive Interview Series. 

During our discussion, we had the privilege of hearing about your career experiences, the growth of ESPN and your role, the greater communications field’s future, and about giving back to the community. Both the Promazo team and our student audience will be excited to learn from you and your experiences.

Attached is a short clip from the interview that we found particularly interesting. The point you made on getting involved with early stage start-ups was insightful for our students searching for early career opportunities.

We are curious to hear if there are any opportunities to conduct similar interviews with some of your colleagues. We have identified the following as potential targets for future interviews: 
Mark Walker
Thomas Hennessy
Laura Gentile
Chris LaPlaca
Chara-Lynn Aguiar

Any contacts for these individuals, or other recommendations you may have, would benefit our project immensely.

Also, if you have any time in the future, we would be thrilled to have you on for a second interview, as your input and advice has been greatly appreciated. A discussion which dives further into your personal journey and visions on the industry’s future would be meaningful to our students.

Again, we thank you for your time and conversation. We look forward to building a closer relationship with you.

**Comment by howie-emond** [comment]
ATTATCHED CLIP
- In Soltys Interview folder
- ESPN-soltys-follow-up.mov

https://drive.google.com/drive/folders/1s0MP6e2F-vmZ1h64faLQh4ydOtg_tVHq?usp=sharing

**Comment by howie-emond** [comment]
Profile Links

Mark Walker
https://www.linkedin.com/in/defmonk/

Thomas Hennessy
https://www.linkedin.com/in/tom-hennessy-4a860930/

Laura Gentile
https://www.linkedin.com/in/gentilelaura/

Chris LaPlaca
https://www.linkedin.com/in/chris-laplaca-59a2325/

Chara-Lynn Aguiar
https://www.linkedin.com/in/chara-lynn-aguiar-67abb96/

**Comment by howie-emond** [comment]
UPDATED MESSAGE

Hi Mike, we wanted to reach out and thank you again for participating in our Executive Interview Series.

During our discussion, we had the privilege of hearing about your career experiences, the growth of ESPN and your role, the greater communications field’s future, and about giving back to the community. Both the Promazo team and our student audience will be excited to learn from you and your experiences.

Attached is a short clip from the interview that we found particularly interesting. The point you made on getting involved with early stage start-ups was insightful for our students searching for early career opportunities.

As we closed off the interview, we discussed getting your advice on our mentorship program. We would greatly appreciate a 15 minute discussion to get your advice and potential involvement. Again, we thank you for your time and conversation, and look forward to building a closer relationship with you.

SAME CLIP
